packages/xdl/src/ProjectSettings.ts

```diff
@@ -75,7 +75,7 @@ export async function readAsync(projectRoot: string): Promise<ProjectSettings> {
   let projectSettings;
   try {
     projectSettings = await projectSettingsJsonFile(projectRoot).readAsync();
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
```suggestion
  } catch {
```
the same comment applies to other changed lines in the file.